packages/bot-engine/package.json

```diff
@@ -0,0 +1,99 @@
+{
+	"name": "@deriv/bot-engine",
```

Comment:
Please fix indentation of this file to match all others (2 spaces per indentation level).